### PR TITLE
feat:Score Page

### PR DESCRIPTION
**Pull request by TheAmlanSaikia**
score page

Featuring Score Page

### CODE REVIEW

File: src/pages/Score Page/Score.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+.score-box{
+    width: 50vw;
+    height: max-content;
+    border-radius: 0.5rem;
+   background-color: #d7faff;
```
Comment (srv-pranjal) [inline src/pages/Score Page/Score.css:6]: You can use variables for colors.

**Review by srv-pranjal — APPROVED** [review]
- Clean and understandable Code
- Good variable naming convention

**Review by aniketsingh13 — APPROVED** [review]
code look clean and good to merge

**Review by ckahub — COMMENTED** [review]
good folder structure
good use of class in css

**Pull request merged**